### ISSUE TITLE
Various case sensitivity issues.

### ISSUE THREAD

**Issue by authanram** [michielkempen/nova-order-field#2]
Just search for your own name and you will discover a lot of typos. 
Let me guess, you work on windows?

Sometimes u use 'michielkempen', sometime 'Michielkempen' and sometimes 'MichielKempen'.

And there is also something wrong with the autoloading. I've fixed all the typos but the Trait can still be not found. Will be back 20 to 50 commits later... good luck.

**Comment by mattlibera** [comment]
I was able to use this simply by correcting the example provided in the README snippet.

Instead of:
```
use MichielKempen\NovaOrderField\Orderable;
use MichielKempen\NovaOrderField\OrderField;
```

use:

```
use Michielkempen\NovaOrderField\Orderable;
use Michielkempen\NovaOrderField\OrderField;
```

That's all that needs to be done here, I think - it's a documentation issue, not a code issue.

**Comment by michielkempen** [comment]
This problem is fixed in version 2.0.0 of the package.
